lib/active_admin/orm/active_record/comments.rb

```
@@ -5,9 +5,9 @@
 
 # Add the comments configuration
 ActiveAdmin::Application.inheritable_setting :comments,                   true
-ActiveAdmin::Application.inheritable_setting :show_comments_in_menu,      true
 ActiveAdmin::Application.inheritable_setting :comments_registration_name, 'Comment'
 ActiveAdmin::Application.inheritable_setting :comments_order,             "created_at ASC"
+ActiveAdmin::Application.inheritable_setting :comments_menu,              false
```

Why is the default set to hide the comments menu?